packages/datagateway-table/src/state/actions/actions.test.tsx

```diff
@@ -54,9 +50,10 @@ describe('Actions', () => {
 
     it('given a empty sort and filters it returns just sorting by ID', () => {
```

Comment:
The rest of these `getApiFilter` tests should live in `datagateway-common` since that's where the function lives and where some of it's tests live

Reply:
done!